msgraph-mail/php/src/Models/Microsoft/Graph/BodyType.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Microsoft\Graph\Models\Microsoft\Graph;
+
+use Microsoft\Kiota\Abstractions\Enum;
+
+class BodyType extends Enum {
```

Comment:
I'm assuming the enum has some kind of implicit parsing and to string capability?

Reply:
Yes that is defined in the base class.